src/libraries/System.IO.Compression.Brotli/src/System/IO/Compression/enc/BrotliStream.Compress.cs

```diff
@@ -132,6 +187,12 @@ public override void Flush()
             }
         }
 
+        /// <summary>Asynchronously clears all buffers for this Brotli stream, causes any buffered data to be written to the underlying device, and monitors cancellation requests.</summary>
```

Comment:
It would be good to see an example of backporting from dotnet-api-docs when there are existing triple slash comments too.

Reply:
Good point to bring up: since right now, dotnet-api-docs is the source of truth, then it doesn't matter if existing triple slash comments get overwritten. If there's anything important we want to keep, then it can be discussed in the PR.

I'll have to add a unit test for this case in my tool anyway, because I think the spacing gets messed up when there are existing comments already.